resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java

```diff
@@ -2767,22 +2766,6 @@ public void testResolveACachedWithDotSearchDomain() throws Exception {
         }
     }
 
-    @Test
-    public void testChannelFactoryException() {
```

Comment:
This is a bit of a change as it not fails fast now anymore but only when we try to resolve. Which I think is better anyway as before we did try to bind directly in the constructor.